lighthouse-plugin-publisher-ads/audits/deprecated-api-usage.js

```diff
@@ -39,20 +41,29 @@ class DeprecatedApiUsage extends Audit {
    */
   static get meta() {
     return {
-      id: 'deprecated-gpt-api-usage',
+      id: 'deprecated-api-usage',
```

Comment:
Nit: I'd say the former name was more descriptive/accurate, and maybe just the filename should be changed instead? Core lighthouse has a generic api deprecation audit that may be confused with this from someone unfamiliar: https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/deprecations.js

Reply:
I'm OK either way. The name is a bit of an implementation detail

Reply:
FYI if you change the audit filename, you also need to change the doc filename. And we'd also need to set up a redirect for any existing links to the old doc.

Reply:
Done!

Reply:
Yeah, for now I only changed the id of the audit and its references.